src/boot/efi/util.c

```diff
@@ -3,69 +3,9 @@
 #include <efi.h>
 #include <efilib.h>
 
+#include "ticks.h"
```

Comment:
It would be better header hygiene to include it where it's actually needed.

Reply:
efivar_set_time_usec() further down in util.c uses time_usec(), that's why we include it here.